painttiming.bs

```diff
@@ -62,9 +86,24 @@ Formally, we consider the user agent to have "rendered" a document when it has p
 
     NOTE: The rendering pipeline is very complex, and the timestamp should be the latest timestamp the user agent is able to note in this pipeline (best effort). Typically the time at which the frame is submitted to the OS for display is recommended for this API.
 
+<dfn export>Contentful</dfn>: An element counts as contentful when one or more of the following apply:
+* The element contains a [=text node=], with at least one character, excluding [=white-space=].
+* The element is a [=replaced image=] representing a [=loaded=] image.
+* The element's current style contains a [=loaded=] [=background-image=] or [=mask-image=].
```

Comment:
Mask image link is also broken (in the sense that it does not point anywhere within that spec). Could you double check all the links you're adding?